c++/build.sh

```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env sh
 clang++ -std=c++11 -Wall -Wextra -Werror -g ./src/linked-list.cc -o out/linked-list.bin
-clang++ -std=c++11 -Wall -Wextra -Werror -g ./src/binary-tree.cc -o out/binary-tree.bin
\ No newline at end of file
+clang++ -std=c++11 -Wall -Wextra -Werror -g ./src/binary-tree.cc -o out/binary-tree.bin
+clang++ -std=c++11 -Wall -Wextra -Werror -g ./src/vector.cc -o out/vector.bin
```

Comment:
Also it might be worth to start looking into either makefiles or cmake (what I use). Here's a [cmake guide](https://cgold.readthedocs.io/en/latest/first-step.html). CMake is cross platform. Using something like one of these (CMake uses makefiles under the hood on linux) will help with for example only building the stuff that changed etc.